fe1-web/model/objects/Chirp.ts

```diff
@@ -7,12 +7,12 @@ import { PublicKey } from './PublicKey';
  */
 
 export interface ChirpState {
-  id: string;
-  sender: string;
-  text: string;
-  time: number;
-  likes: number;
-  parentId: string;
+  readonly id: string;
+  readonly sender: string;
+  readonly text: string;
+  readonly time: number;
+  readonly likes: number;
+  readonly parentId: string;
```

Comment:
To be honest I don't know if they are necessary. There were some weird behaviours then I saw in model/objects/LaoEvent they add 'readonly' for state interface so I add them here. In the end, the problem wasn't because of this, and I forgot to remove them.